sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -408,11 +413,26 @@ def analyze_sentiment(  # type: ignore
         docs = _validate_batch_input(documents, "language", language)
         model_version = kwargs.pop("model_version", None)
         show_stats = kwargs.pop("show_stats", False)
+        show_aspects = kwargs.pop("show_aspects", None)
+
         try:
+            if self._api_version == "v3.0":
```

Comment:
is this how we plan to handle new keywords moving forward?

Reply:
Ideally the autorest generated multiapi client will handle it, the checking for the keywords should happen there but since it can't handle that yet, I have this code for now

Reply:
should we make an issue to update this code once autorest multiapi client can handle it?

Reply:
good idea, thanks!